README.md

```diff
@@ -73,12 +75,28 @@ return [
     ],
 ];
 ```
-
 The server token can be found in your Postmark account.
 
+#### Amazon SES
+In order to configure Amazon SES, add the following code to your config:
+```php
+return [
+    'zfr_mail' => [
+        'aws_ses' => [
```

Comment:
I'd recommend you to make this package with an optional dependency to https://github.com/zf-fr/zfr-aws-utils

This is actually a super lightweight library that integrates with AWS, and allows you to set that kind of config:

```php
return [
   'aws' => [
       // AWS config
   ]
]
```

and then allow to get a client:

```php
$sdk = $container->get(Aws\Sdk::class);
$ses = $sdk->createSesClient();
```

This has the advantage to centralize all AWS config into a single config key and therefore avoid having it defined for each library that uses a client.

So I suggest:

* Add this library as a "suggested" library
* Update documentation to indicate that SES requires usage of this library
* Update doucmenttaion for the config

What do you think? :)

Reply:
I think it's good but ZfrAwsUtils has dependency to Zend Service Manager ><

Reply:
ZfrAwsUtils has no required dependency against Zend Service Manager: https://github.com/zf-fr/zfr-aws-utils/blob/master/composer.json#L22 and the factory to create the client is 100% PSR compliant so can be re-used with anything: https://github.com/zf-fr/zfr-aws-utils/blob/master/src/Container/SdkFactory.php :)

Reply:
Ok ok I made the change :p